packages/streamline/src/components/navigation/navigation-title/navigation-title.tsx

```diff
@@ -0,0 +1,87 @@
+import { Box } from '../../../primitives/box/box';
+import { Text } from '../../../primitives/text/text';
+import { IconsName } from '../../../primitives/icon/icon.types';
+import ButtonIcon from '../../buttons/button-icon/button-icon';
+import { getNavigationTitleColors } from './navigation-title.utils';
+
+export interface NavigationTitleProps {
+  appearance?: 'dark' | 'light';
+  position?: 'left' | 'center';
+  title: string;
+  header?: string;
+  value?: string;
+  icon?: IconsName;
+  onIconPress?: () => void;
+  buttonIconAccessibilityLabel?: string;
+}
+
+export const NavigationTitle = ({
+  appearance = 'dark',
+  position = 'left',
+  title,
+  header,
+  value,
+  icon,
+  onIconPress,
+  buttonIconAccessibilityLabel,
+}: NavigationTitleProps) => {
+  const { titleColor, headerColor, valueColor, headerOpacity, valueOpacity } =
+    getNavigationTitleColors(appearance);
+
+  return (
+    <Box
+      paddingHorizontal={'md'}
+      paddingVertical={'xs'}
+      alignItems={position === 'left' ? 'flex-start' : 'center'}
+    >
+      {header && (
```

Comment:
User the ternary `? : null` to prevent some rendering issues. For example, if `header` is an **empty string**, it will render but raise an error because text will be try to be rendered outside `Text` component.

Reply:
I saw that it's not currently part of our guidelines, would you have any objection if we add it?

Reply:
Good catch! And that's part of the guideline here:
https://www.notion.so/luko/React-Native-Bible-Guidelines-8fe341be987d4cb88185de5db10e71f9?pvs=4#475d24968d224c63a58e0d3cb0c78358
ideally, we should find a way to automate it.